core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -57,6 +57,12 @@ class OpenMPInternal {
       : m_pool_size{arg_pool_size}, m_level{omp_get_level()}, m_pool() {
     // guard pushing to all_instances
     {
+#ifndef KOKKOS_ENABLE_DEPRECATED_CODE_4
```

Comment:
Please briefly speak to this being proposed as deprecated code 4 and not deprecated code 5 (as a reply to this comment)
(Just asking for more information at this time)